backend/tests/test_accounts.py

```diff
@@ -219,3 +219,77 @@ def test_account_balance_missing_on_creation(
     )
     assert res.status_code == 201
     assert res.json["account"]["account_balance"] == 0
+
+
+@pytest.mark.parametrize(
+    "account_id, account_type, account_name, account_balance, username",
+    TEST_NEW_ACCT_DATA,
+)
+def test_get_account_success(
+    client, account_id, account_type, account_name, account_balance, username
+):
+    client.post("/user", json={"username": username, "first_name": "", "last_name": ""})
+    post_res = client.post(
+        "/account",
+        json={
+            "account_id": account_id,
+            "account_type": account_type,
+            "account_name": account_name,
+            "account_balance": account_balance,
+            "username": username,
+        },
+    )
+    acct_id = post_res.json["account"]["account_id"]
+    res = client.get(f"/account?account_id={acct_id}&username={username}")
```

Comment:
Consider modifying the required query parameters for the get request. The `account_id` sufficiently maps to a specific account. To query for all accounts a user may own may be a separate endpoint that looks like `/account?username=xxx`

Reply:
created placeholder for future functionality